# Spanish translation for linuxmint
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the linuxmint package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: linuxmint\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2016-04-22 13:59+0100\n"
"PO-Revision-Date: 2016-06-19 21:23+0000\n"
"Last-Translator: Isidro Pisa <isidro@utils.info>\n"
"Language-Team: Spanish <es@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2016-06-25 09:26+0000\n"
"X-Generator: Launchpad (build 18111)\n"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:111
#: usr/lib/linuxmint/mintbackup/mintbackup.py:252
#: usr/lib/linuxmint/mintbackup/mintbackup.py:449
#: usr/lib/linuxmint/mintbackup/mintbackup.py:467
#: usr/lib/linuxmint/mintbackup/mintbackup.py:533
#: usr/lib/linuxmint/mintbackup/mintbackup.py:536
#: usr/lib/linuxmint/mintbackup/mintbackup.py:588
#: usr/lib/linuxmint/mintbackup/mintbackup.py:591
#: usr/lib/linuxmint/mintbackup/mintbackup.py:611
#: usr/lib/linuxmint/mintbackup/mintbackup.py:634
#: usr/lib/linuxmint/mintbackup/mintbackup.py:641
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1498
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1596
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1604
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1656 generate_desktop_files.py:26
#: generate_desktop_files.py:41 generate_desktop_files.py:56
msgid "Backup Tool"
msgstr "Herramienta de copia de seguridad"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:169
msgid "Preserve structure"
msgstr "Conservar estructura"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:171
msgid ".tar file"
msgstr "archivo .tar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:172
msgid ".tar.bz2 file"
msgstr "archivo .tar.bz2"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:173
msgid ".tar.gz file"
msgstr "archivo .tar.gz"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:179
msgid "Never"
msgstr "Nunca"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:180
msgid "Size mismatch"
msgstr "El tamaño no coincide"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:181
msgid "Modification time mismatch"
msgstr "La fecha de modificación no coincide"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:182
msgid "Checksum mismatch"
msgstr "La suma de comprobación no coincide"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:183
msgid "Always"
msgstr "Siempre"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:205
msgid "Excluded paths"
msgstr "Rutas excluidas"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:215
msgid "Type"
msgstr "Tipo"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:219
msgid "Detail"
msgstr "Detalle"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:225
#: usr/lib/linuxmint/mintbackup/mintbackup.py:234
msgid "Path"
msgstr "Ruta"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:228
#: usr/lib/linuxmint/mintbackup/mintbackup.py:237
msgid "Error"
msgstr "Error"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:269
msgid "Store?"
msgstr "¿Almacenar?"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:272
#: usr/lib/linuxmint/mintbackup/mintbackup.py:285
msgid "Name"
msgstr "Nombre"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:282
msgid "Install"
msgstr "Instalar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:290
msgid "Backup or Restore"
msgstr "Respaldar o restaurar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:291
msgid "Choose from the following options"
msgstr "Elija entre las siguientes opciones"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:293
#: usr/lib/linuxmint/mintbackup/mintbackup.py:309
#: usr/lib/linuxmint/mintbackup/mintbackup.py:323
#: usr/lib/linuxmint/mintbackup/mintbackup.py:330
#: usr/lib/linuxmint/mintbackup/mintbackup.py:334
#: usr/lib/linuxmint/mintbackup/mintbackup.py:339
msgid "Backup files"
msgstr "Respaldar archivos"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:294
#: usr/lib/linuxmint/mintbackup/mintbackup.py:343
#: usr/lib/linuxmint/mintbackup/mintbackup.py:353
#: usr/lib/linuxmint/mintbackup/mintbackup.py:359
#: usr/lib/linuxmint/mintbackup/mintbackup.py:364
msgid "Restore files"
msgstr "Restaurar archivos"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:295
#: usr/lib/linuxmint/mintbackup/mintbackup.py:368
#: usr/lib/linuxmint/mintbackup/mintbackup.py:373
#: usr/lib/linuxmint/mintbackup/mintbackup.py:379
#: usr/lib/linuxmint/mintbackup/mintbackup.py:384
msgid "Backup software selection"
msgstr "Respaldar selección de software"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:296
#: usr/lib/linuxmint/mintbackup/mintbackup.py:388
#: usr/lib/linuxmint/mintbackup/mintbackup.py:393
#: usr/lib/linuxmint/mintbackup/mintbackup.py:400
msgid "Restore software selection"
msgstr "Restaurar selección de software"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:298
msgid "Make a backup of your files"
msgstr "Hace un respaldo de sus archivos"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:299
msgid "Save the list of installed applications"
msgstr "Guarda la lista de aplicaciones instaladas"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:300
msgid "Restore a previous backup"
msgstr "Restaura un respaldo anterior"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:301
msgid "Restore previously installed applications"
msgstr "Restaura aplicaciones instaladas previamente"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:310
msgid "Please select a source and a destination for your backup"
msgstr "Seleccione el origen y destino para su respaldo"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:311
#: usr/lib/linuxmint/mintbackup/mintbackup.py:347
#: usr/lib/linuxmint/mintbackup/mintbackup.py:355
msgid "Source:"
msgstr "Origen:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:312
#: usr/lib/linuxmint/mintbackup/mintbackup.py:348
msgid "Destination:"
msgstr "Destino:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:313
#: usr/lib/linuxmint/mintbackup/mintbackup.py:349
msgid "Advanced options"
msgstr "Opciones avanzadas"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:314
#: usr/lib/linuxmint/mintbackup/mintbackup.py:356
msgid "Description:"
msgstr "Descripción:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:315
msgid "Output:"
msgstr "Salida:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:316
#: usr/lib/linuxmint/mintbackup/mintbackup.py:350
msgid "Overwrite:"
msgstr "Sobreescribir:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:317
msgid "Confirm integrity"
msgstr "Confirmar integridad"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:318
msgid "Follow symlinks"
msgstr "Seguir enlaces simbólicos"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:319
msgid "Preserve permissions"
msgstr "Conservar permisos"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:320
msgid "Preserve timestamps"
msgstr "Conservar marcas de tiempo"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:324
msgid "Please select any files or directories you want to exclude"
msgstr "Seleccione los archivos y carpetas que desea excluir"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:325
msgid "Exclude files"
msgstr "Excluir archivos"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:326
msgid "Exclude directories"
msgstr "Excluir carpetas"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:327
msgid "Remove"
msgstr "Eliminar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:331
msgid "Please review the information below before starting the backup"
msgstr "Revise la siguiente información antes de empezar el respaldo"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:335
msgid "Please wait while your files are being backed up"
msgstr "Por favor, espere mientras se respaldan sus archivos"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:336
#: usr/lib/linuxmint/mintbackup/mintbackup.py:381
msgid "Backing up:"
msgstr "Respaldando:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:340
#: usr/lib/linuxmint/mintbackup/mintbackup.py:385
msgid "The backup is now finished"
msgstr "El respaldo ha finalizado"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:344
msgid ""
"Please choose the type of backup to restore, its location and a destination"
msgstr "Elija el tipo de respaldo a restaurar, su ubicación y un destino"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:345
msgid "Archive"
msgstr "Archivador"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:346
msgid "Directory"
msgstr "Carpeta"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:354
msgid "Please review the information below"
msgstr "Revise la siguiente información"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:360
msgid "Please wait while your files are being restored"
msgstr "Por favor, espere mientras se restauran sus archivos"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:361
msgid "Restoring:"
msgstr "Restaurando:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:365
msgid "The restoration of the files is now finished"
msgstr "La restauración de los archivos ha finalizado"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:369
msgid "Please choose a destination"
msgstr "Seleccione un destino"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:370
#: usr/lib/linuxmint/mintbackup/mintbackup.py:552
msgid "Destination"
msgstr "Destino"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:374
msgid "The list below shows the packages you added to Linux Mint"
msgstr "La siguiente lista muestra los paquetes que añadió a Linux Mint"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:375
#: usr/lib/linuxmint/mintbackup/mintbackup.py:395
msgid "Select all"
msgstr "Seleccionar todos"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:376
#: usr/lib/linuxmint/mintbackup/mintbackup.py:396
msgid "Deselect all"
msgstr "Deseleccionar todos"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:380
msgid "Please wait while your software selection is being backed up"
msgstr "Por favor, espere mientras se respalda su selección de software"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:389
msgid "Please select a saved software selection"
msgstr "Elija una selección de software guardada"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:390
msgid "Software selection:"
msgstr "Selección de software:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:394
msgid "Select the packages you want to install"
msgstr "Seleccione los paquetes que quiere instalar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:397
msgid "Refresh"
msgstr "Actualizar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:401
msgid "The restoration is now finished"
msgstr "La restauración ha finalizado"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:402
msgid "Your package selection was restored successfully"
msgstr "Su selección de paquetes se ha restaurado con éxito"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:458
#: usr/lib/linuxmint/mintbackup/mintbackup.py:476
msgid "Invalid path"
msgstr "Ruta no válida"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:458
#: usr/lib/linuxmint/mintbackup/mintbackup.py:476
#, python-format
msgid "%s is not located within your source directory. Not added."
msgstr "%s no se encuentra dentro de su carpeta de origen. No se añadió."

#: usr/lib/linuxmint/mintbackup/mintbackup.py:533
msgid "Please choose directories for the source and the destination"
msgstr "Por favor, elija las carpetas para el origen y el destino"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:536
msgid ""
"Please choose different directories for the source and the destination"
msgstr "Por favor, elija carpetas diferentes para el origen y el destino"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:551
msgid "Source"
msgstr "Origen"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:554
msgid "Description"
msgstr "Descripción"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:558
msgid "Compression"
msgstr "Compresión"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:562
msgid "Overwrite destination files"
msgstr "Sobrescribir archivos de destino"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:565
msgid "Exclude"
msgstr "Excluir"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:588
msgid "Please choose a file to restore from"
msgstr "Seleccione un archivo desde el que restaurar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:591
#: usr/lib/linuxmint/mintbackup/mintbackup.py:611
msgid "Please choose a destination directory"
msgstr "Por favor, seleccione una carpeta de destino"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:634
msgid "No packages need to be installed at this time"
msgstr "No es necesario instalar paquetes en este momento"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:641
msgid "Please select one or more packages to install"
msgstr "Por favor, seleccione uno o más paquetes para instalar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:709
#: usr/lib/linuxmint/mintbackup/mintbackup.py:710
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1204
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1206
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1513
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1514
msgid "Calculating..."
msgstr "Calculando..."

#: usr/lib/linuxmint/mintbackup/mintbackup.py:759
#: usr/lib/linuxmint/mintbackup/mintbackup.py:799
msgid "Skipping broken link"
msgstr "Omitiendo enlace roto"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:760
msgid "Broken link"
msgstr "Enlace roto"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:763
#: usr/lib/linuxmint/mintbackup/mintbackup.py:803
msgid "Skipping link"
msgstr "Omitiendo enlace"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:827
#: usr/lib/linuxmint/mintbackup/mintbackup.py:836
#: usr/lib/linuxmint/mintbackup/mintbackup.py:845
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1265
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1276
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1289
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1354
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1363
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1373
msgid "Skipping identical file"
msgstr "Omitiendo archivo idéntico"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:867
#, python-format
msgid ""
"Warning: Some files were not saved, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""
"Aviso: algunos archivos no se guardaron; copiados: %(current_file)d archivos "
"de un total de %(total)d"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:871
msgid "An error occurred during the backup"
msgstr "Ha ocurrido un error durante la copia de seguridad"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:881
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1559
msgid "The backup was aborted"
msgstr "El respaldo fue cancelado"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:889
msgid "The backup completed successfully"
msgstr "El respaldo se completó correctamente"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:980
msgid "Checksum Mismatch:"
msgstr "La suma de comprobación no coincide:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:981
msgid "Checksum Mismatch"
msgstr "La suma de comprobación no coincide"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1033
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1035
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1064
msgid "Calculating checksum"
msgstr "Calculando suma de comprobación"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1110
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1134
msgid "(Not created with the Backup Tool)"
msgstr "(No creado con la herramienta de respaldo)"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1397
#, python-format
msgid ""
"Warning: Some files were not restored, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""
"Atención: Algunos archivos no se restauraron. Copiados: %(current_file)d "
"archivos de %(total)d en total"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1400
msgid "An error occurred during the restoration"
msgstr "Ha ocurrido un error durante la restauración"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1411
msgid "The restoration was aborted"
msgstr "La restauración fue cancelada"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1419
msgid "The restoration completed successfully"
msgstr "La restauración se completó correctamente"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1550
msgid "An error occurred during the backup:"
msgstr "Ocurrió un error durante la copia de seguridad:"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1567
msgid "Your software selection was backed up successfully"
msgstr "Su selección de software se ha respaldado con éxito"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1596
msgid "The specified file is not a valid software selection"
msgstr "El archivo especificado no es una selección de software válida"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1604
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1656
msgid "An error occurred while accessing the file"
msgstr "Ocurrió un error mientras se accedía al archivo"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1649
msgid "Could not locate the package"
msgstr "No se pudo localizar el paquete"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1681
msgid "Please wait, this can take some time"
msgstr "Espere, por favor; esto puede tardar algún tiempo"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1683
msgid "The installation is complete"
msgstr "La instalación ha finalizado"

#: generate_desktop_files.py:26 generate_desktop_files.py:41
#: generate_desktop_files.py:56
msgid "Make a backup of your home directory"
msgstr "Hacer una copia de seguridad de su carpeta personal"

#~ msgid "Cleaning up..."
#~ msgstr "Limpiando..."

#~ msgid "Exclude folders"
#~ msgstr "Excluir carpetas"

#~ msgid "Opening the backup archive..."
#~ msgstr "Abriendo la copia de seguridad..."

#~ msgid "Read error"
#~ msgstr "Error de lectura"

#~ msgid "Hidden paths"
#~ msgstr "Rutas ocultas"

#~ msgid "Included hidden directories"
#~ msgstr "Directorios ocultos incluidos"

#~ msgid "Included"
#~ msgstr "Incluido"

#~ msgid "Restore"
#~ msgstr "Restaurar"

#~ msgid "View content"
#~ msgstr "Visualizar contenido"

#~ msgid "Restoration successful"
#~ msgstr "Restauración exitosa"

#~ msgid "<big><b>Load data into your home directory</b></big>"
#~ msgstr "<big><b>Cargar datos en su directorio personal</b></big>"

#~ msgid "is not located within your home directory. Not added."
#~ msgstr "no se encuentra en su directorio personal. No se ha añadido."

#~ msgid "Excluded Files and Directories"
#~ msgstr "Archivos y directorios excluidos"

#~ msgid "Restoration failed"
#~ msgstr "Restauración fallida"

#~ msgid "An error occurred while restoring the backup archive:"
#~ msgstr "Ocurrió un error al restaurar la copia de seguridad:"

#~ msgid "Backup failed"
#~ msgstr "La copia de seguridad falló"

#~ msgid "Extracting the content of the backup into your home directory..."
#~ msgstr ""
#~ "Extrayendo el contenido de la copia de seguridad en su carpeta personal..."

#~ msgid "Archiving your home directory..."
#~ msgstr "Archivando su carpeta personal..."

#~ msgid "Backup"
#~ msgstr "Respaldo"

#~ msgid "Backup successful"
#~ msgstr "Copia de seguridad realizada"

#~ msgid "Your home directory was successfully backed-up into"
#~ msgstr "Su carpeta personal se copió con éxito en"

#~ msgid "Copying the backup file into your home directory..."
#~ msgstr "Copiando el archivo de copia de seguridad en su carpeta personal..."

#~ msgid "Your backup was successfully restored"
#~ msgstr "Su copia de seguridad se restauró correctamente"

#~ msgid "Overwrite existing files"
#~ msgstr "Sobrescribir archivos existentes"

#~ msgid "Restore your personal data from this backup"
#~ msgstr "Restaurar sus datos personales a partir de este respaldo"

#~ msgid "An error occurred while opening the backup:"
#~ msgstr "Se ha producido un error al abrir el respaldo:"
